src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/PyrisConnectorService.java

```diff
@@ -79,6 +80,20 @@ void executePipeline(String feature, String variant, Object executionDTO) {
         }
     }
 
+    void executeWebhook(String variant, PyrisWebhookLectureIngestionExecutionDTO executionDTO) {
+        var endpoint = "/api/v1/webhooks/" + variant;
+        try {
+            restTemplate.postForEntity(pyrisUrl + endpoint, objectMapper.valueToTree(executionDTO), Void.class);
+        }
+        catch (HttpStatusCodeException e) {
+            throw toIrisException(e);
+        }
+        catch (RestClientException | IllegalArgumentException e) {
+            log.error("Failed to send lectures to Pyris", e);
+            throw new PyrisConnectorException("Could not fetch response from Iris");
+        }
+    }
+
```

Comment:
This should be more specifically for lectures:
- Name it `executeLectureWebhook`
- Make the URL `/api/v1/webhooks/lectures/<variant>`
-